modules/mfa/src/Auth/Process/Mfa.php

```diff
@@ -289,6 +291,33 @@ public static function getNumBackupCodesUserHad(array $mfaOptions): int
         return $numBackupCodes;
     }
 
+    /**
+     * @throws GuzzleException
+     */
+    public static function getRecoveryContacts(?array $state): array
+    {
+        $recoveryConfig = $state['recoveryConfig'] ?? [];
+        $emailAddressValues = $state['Attributes']['mail'] ?? [''];
+        $emailAddress = $emailAddressValues[0] ?? '';
+
+        $apiUrl = $recoveryConfig['api'] ?? '';
+        $apiKey = $recoveryConfig['apiKey'] ?? '';
+        $fallbackEmail = $recoveryConfig['fallbackEmail'] ?? '';
+        $fallbackName = $recoveryConfig['fallbackName'] ?? '';
+
+        $apiClient = new ApiClient($apiKey);
```

Comment:
Would it be helpful to describe what API this is calling? Either a more specific variable name or a comment or both? I _think_ I know, but I'm not sure. Since this is a public repo and the other is not, maybe a name like `recoveryApiClient` is enough. The README has a bit of documentation on it, though it doesn't describe what the request or response should contain.

Reply:
Thanks for sharing that. I'll rename some variables here, to hopefully make it clearer.

As for the readme... I'm a bit confused, because I did (attempt to) specify there how the request works and what structure the response needs to have ,here:  
<https://github.com/silinternational/ssp-base?tab=readme-ov-file#new-recovery-contacts-feature-coming-soon>

Please feel free to share any additional details that you would find helpful to have in the readme file about this.

Reply:
@briskt I'm going to go ahead and resolve this for now (so I can proceed with the merge, since you approved the PR). Feel free to continue this conversation, though, and I can make additional requested changes in future PRs.

Reply:
I'm sorry. I didn't read the README carefully enough. You had good documentation already.